src/ocl/tensorocl.cpp

```diff
@@ -344,40 +340,17 @@ void OpTensor3d(const Handle& handle,
                 long(Aoffset),
                 long(Boffset),
                 long(Coffset),
-                int(clens[1]));
+                long(total_work),
+                long(total_work2),
+                int(!float_equal(miopen_beta, 0.0)),
```

Comment:
could we have used short in place of int to pass bool?

Reply:
https://github.com/ROCmSoftwarePlatform/MIOpen/pull/349#discussion_r456717071